webapp/src/clj/lipas/ai/core.clj

```diff
@@ -110,6 +132,8 @@ Provide answers in English, Finnish, and Swedish. Different language versions ca
 
 (defn ->prompt-doc
   [sports-site]
+  ;; Might include (some) of the UTP data now?
+  ;; Could be a good thing, but might make the prompt data too large?
```

Comment:
It actually looks like I've considered UTP data here.. Since `dissoc` contains `videos` and `images` that are clearly UTP stuff. However, it might be good to review if there are keys that:

- don't provide clear value for text generation
- can make the payload too large for the AI

I think `:ptv` should be dissoc'd also... since it contains only identifiers and previously generated texts.

Also, some of the UTP-fields may contain very long texts...